src/components/ChatComponents/ChatIcons.tsx

```diff
@@ -188,15 +153,15 @@ const ChatIcons: React.FC<ChatIconsProps> = ({
           <span className="tooltip-text">Model Selection</span>
         </div>
       </div>
-      <button className="chat-icon-button" onClick={onNewChat}>
+      <button className="chat-icon-button clickable-icon" onClick={onNewChat}>
```

Comment:
I don't see how `clickable-icon` is used? Maybe I'm missing something.

Reply:
That's to prevent the buttons from having extra padding. It's defined in the obsidian `app.css`:
<img width="1102" alt="image" src="https://github.com/user-attachments/assets/08a4720e-29e9-4919-bd0d-86396c384daa">

Reply:
Was the "extra padding" looking weird on mobile? because with this change it looks weirder to me, they are now big icons and don't share the consistent button look with others.

Now: 
<img width="449" alt="SCR-20240829-ooes" src="https://github.com/user-attachments/assets/3ea82734-648c-4d3b-8368-3443c9cec51d">

Before (looks better IMO):
<img width="437" alt="SCR-20240829-ooiv" src="https://github.com/user-attachments/assets/deb7fac1-bbff-4fcd-b98c-6dd0e895ec40">